systemservices/service.go

```diff
@@ -0,0 +1,111 @@
+package systemservices
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"sync"
+
+	"github.com/docker/docker/pkg/archive"
+	"github.com/mesg-foundation/core/service"
+	"github.com/mesg-foundation/core/x/xerrors"
+)
+
+// deploySystemServices deploys system services placed inside the system services path.
+func (s *SystemServices) deploySystemServices() ([]*service.Service, error) {
```

Comment:
I would recommend to implement a simpler logic in this function.
As `SystemServices` knows all `SystemService` (name, path and type. cf https://forum.mesg.com/t/plugin-system-use-mesg-service-to-develop-mesg-core/23/23), this function could simply access the predefined path of each service.
For Resolver, as its name and path is `resolver`, it could try to access directly `filepath.Join(s.systemServicesPath, name)` (`~/.mesg/systemservices/resolver/`).

Reply:
And don't forget to use from Config the value of `config.Core.Path` to build the filepath.
The absolute filepath is:
`filepath.Join(config.Core.Path, config.SystemServices.RelativePath, systemServiceName)`

Reply:
done